relying-party/src/components/Header.jsx

```diff
@@ -0,0 +1,14 @@
+import './Header.css'
+
+function Header() {
+    return (
+        <header>
```

Comment:
You could put an id on this element such as "page-header" since it should be the only page header.

Reply:
resolved issues